Pod/Classes/OTReference.h

```diff
@@ -0,0 +1,45 @@
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
By using this, is this mandating that consumers of OpenTracing be on a relatively new version of XCode? Is that an okay constraint to enforce? Or to put it differently, if this *isn't* used in this file, will the library be more likely to "just work" for more people?